notation.go

```diff
@@ -325,3 +390,17 @@ func generateAnnotations(signerInfo *signature.SignerInfo) (map[string]string, e
 		annotationX509ChainThumbprint: string(val),
 	}, nil
 }
+
+func (outcome *VerificationOutcome) GetUserMetadata() (map[string]string, error) {
+	var payload envelope.Payload
+	err := json.Unmarshal(outcome.EnvelopeContent.Payload.Content, &payload)
```

Comment:
In case of a bad outcome, `outcome.EnvelopeContent` can be `nil` and thus will panic.